modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc

```diff
@@ -13,18 +13,13 @@ Connected Apps help users delegate their access without sharing sensitive creden
 
 == Creating a Connected App
 
-To create a Connected App, see xref:access-management::connected-apps-developers.adoc[].
+To create a Connected App, see xref:access-management::connected-apps-developers.adoc[]
```

Comment:
```suggestion
To create a connected app, see xref:access-management::connected-apps-developers.adoc[].
```

Reply:
We use lowercase when referring to the thingie generically; init cap for the feature name.